pkg/controller/replicaset/replica_set_test.go

```diff
@@ -680,131 +564,6 @@ func TestWatchPods(t *testing.T) {
 	}
 }
 
-func TestUpdatePods(t *testing.T) {
```

Comment:
Let's not remove this yet. It's not covered by integration. We need to refactor this into a unit test around a function that just outputs which RSes should be enqueued for a given Pod update event. Can you add it to the tracking issue for unit test refactors?

Reply:
sure!